spec/timezone.html

```diff
@@ -865,7 +865,6 @@ <h1>
         1. If _parseResult_ is a List of errors, throw a *RangeError* exception.
         1. If _parseResult_ contains a |TimeZoneIANAName| Parse Node, then
           1. Let _name_ be the source text matched by the |TimeZoneIANAName| Parse Node contained within _parseResult_.
-          1. NOTE: _name_ is syntactically valid, but does not necessarily conform to <a href="https://data.iana.org/time-zones/theory.html#naming">IANA Time Zone Database naming guidelines</a> or correspond with an available named time zone identifier.
```

Comment:
Why is this being removed?

Reply:
Hmm, weird. Looks like a merge conflict resolution gone awry. Will be fixed in the next push.